app/routers/auth.py

```diff
@@ -5,18 +5,23 @@
 import os
 from typing import Annotated
 from datetime import timedelta
-from fastapi import Depends, HTTPException, status, APIRouter
+from fastapi import Depends, HTTPException, status, APIRouter, Response
 from fastapi.security import OAuth2PasswordRequestForm
+from mongoengine.errors import NotUniqueError
 
 router = APIRouter(
   prefix="/auth",
-)   
+)
 
 ACCESS_TOKEN_EXPIRE_MINUTES = int(os.getenv("ACCESS_TOKEN_EXPIRE_MINUTES"))
 
+def setCookie(response, value):
+    response.set_cookie(key="code_names_access_token", value=value, secure=True, samesite="None", max_age=ACCESS_TOKEN_EXPIRE_MINUTES)
+
```

Comment:
Je voulais laisser le router pour uniquement les routes a l'origine, tu penses pouvoir déplacer la définition de la fonction dans le authServices ?

Reply:
if you want...